WeatherMonitoringAndReportingService/Bots/SunBot.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WeatherMonitoringAndReportingService.Config;
+using WeatherMonitoringAndReportingService.WeatherDetails;
+
+namespace WeatherMonitoringAndReportingService.Bots;
+
+public class SunBot: IBot
+{
+    private WeatherConfigurationService _weatherConfigurationService;
+
+    public SunBot(WeatherConfigurationService weatherConfigurationService)
+    {
+        _weatherConfigurationService = weatherConfigurationService;
+    }
```

Comment:
Replace with an interface called `IWeatherConfigurationService` that implements the `GetBotConfiguration` and `UpdateBotConfiguration` methods for it to be injected into the bot classes.